### PR TITLE
Final paper updates

### PR DESCRIPTION
**Pull request by breezykermo**

### CODE REVIEW

File: final-paper/final-paper.org [hunk 1]
```diff
@@ -61,22 +60,23 @@ Since vectors with the greatest similarity may not satisfy the predicate, this a
...
+In scenarios like this, a general-purpose predicate-agnostic index can be supplemented by carefully chosen supplemental indices that are constructed so as to effectively serve some frequently occurring search predicates.
+Intelligently routing to supplemental indexes in such cases is the goal of OAK.
 
 * Background and Related Work
 
-The underlying data structure of both ACORN and OAK is Hierarchical Navigable Small Worlds (HNSW)[cite:https://arxiv.org/abs/1603.09320], a hierarchical, tree-link structure where each node of the tree represents a set of vectors.
-More specifically, it is a /proximity graph/, in which two vertices are linked based on proximity.
+The underlying data structure that enables vector similiarity search in both ACORN and OAK is Hierarchical Navigable Small Worlds (HNSW) [cite:@malkovEfficientRobustApproximate2018], an index that makes use of a hierarchical, tree-link structure during search to more quickly work through the plausible search space while retaining reasonable recall.
+More specifically, HNSW leverages a /proximity graph/, in which two vertices are linked based on proximity.
```
Comment (breezykermo) [inline final-paper/final-paper.org:79]: @csirianni what are you referring to here when talking about HNSW's proximity graph?

File: final-paper/final-paper.org [hunk 2]
```diff
@@ -61,28 +60,30 @@ Since vectors with the greatest similarity may not satisfy the predicate, this a
...
 Proximity is usually computed using Euclidean distance, though other similarity metrics exist (e.g. cosine similarity).
 
 ACORN modifies the HNSW construction algorithm to use neighbor expansion, creating a denser graph.
 While HNSW collects $M$ approximate nearest neighbors as candidate edges for each node in the index, ACORN collects $M \dot \gamma$ approximate nearest neighbors as candidate edges per node.
 The intuition is that given enough redundant nodes, the search space is sufficiently large, even when filtering based on the predicate during search.
```
Comment (breezykermo) [inline final-paper/final-paper.org:84]: I think we could use a finer grained description of ACORN and its neighbor expansion here. (ACORN can /either/ make a denser graph, ACORN-gamma, or can do the expansion at search time, ACORN-1 -- if I remember correctly.)

File: final-paper/final-paper.org [hunk 3]
```diff
@@ -61,28 +60,30 @@ Since vectors with the greatest similarity may not satisfy the predicate, this a
...
 
+
 This is not always the case, though. 
 If the predicate selectivity falls below a minimum specified threshold, ACORN resorts to pre-filtering and brute force search, favoring recall over performance.
```
Comment (breezykermo) [inline final-paper/final-paper.org:88]: Is there a citation for this? I don't remember this from the paper.

File: final-paper/final-paper.org (hunk 1 above)
Comment (csirianni) [reply final-paper/final-paper.org:90]: https://www.pinecone.io/learn/series/faiss/hnsw/

> HNSW slots into the graph category. More specifically, it is a proximity graph, in which two vertices are linked based on their proximity (closer vertices are linked) — often defined in Euclidean distance.

File: final-paper/final-paper.org (hunk 3 above)
Comment (csirianni) [reply final-paper/final-paper.org:99]: In Section 5.2, 

> One simple choice for $\gamma$ is 1 $s_min$, where $s_min$ is the minimum predicate selectivity we plan to
serve before resorting to pre-filtering.

File: final-paper/final-paper.org (hunk 2 above)
Comment (csirianni) [reply final-paper/final-paper.org:95]: Added a TODO.

**Pull request merged**